docs/src/index.md

```diff
@@ -12,13 +12,36 @@ TrixiParticles.jl is a numerical simulation framework designed for particle-base
   - VTK
 
 ## Examples
-
+```@raw html
+<table align="center" border="0">
+  <tr>
+    <td align="center">
+      <img src="https://github.com/trixi-framework/TrixiParticles.jl/assets/10238714/683e9363-5705-49cc-9a5c-3b47d73ea4b8" style="width: 80% !important;"/><br><figcaption>2D Dam Break</figcaption>
+    </td>
+    <td align="center">
+      <img src="https://github.com/trixi-framework/TrixiParticles.jl/assets/10238714/c10faddf-0400-47c9-b225-f5d286a8ecb8" style="width: 80% !important;"/><br><figcaption>Moving Wall</figcaption>
```

Comment:
I have not created a picture of the 3D version.

Reply:
I was thinking about the 2D version, but I guess it would be great to have a 3D simulation on the landing page.
Then leave it like this for now, and I'll add a 3D picture that I have lying around.